build/bump_version.ps1

```diff
@@ -81,6 +81,60 @@ $NEXT_VERSION_TAG = "v" + "$NEXT_VERSION"
 $RELEASE_DATE = (Select-String -Pattern "\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])" -Path $CHANGELOG_PATH | Select-Object -First 1).Matches.Value
 $RELEASE_NOTE_LINK = $NEXT_VERSION.Replace(".", "") + "-" + "$RELEASE_DATE"
 
+###########################################################################
+# Reorder changelog sections
+###########################################################################
+
+$sections = @()
+$sections += ('### ⚠ BREAKING CHANGES')
+foreach($line in Get-Content $VERSIONRC_PATH) {
+    $found = $line -match '(?<=section": ").*(?=",)'
+    if ($found) {
+        $sections += $matches[0]
+    }
+}
+
+$orderedSections = new-object string[] $sections.Length
+$currentSection = $null
+$previousSectionIndex = 0
+
+foreach($line in Get-Content $CHANGELOG_PATH) {
+    if($line -match "## [[0-9]+\.[0-9]+\.[0-9]+]"){
```

Comment:
It should be "#{2,3} [[0-9]+\.[0-9]+\.[0-9]+]" because if there is only release with fixes then # character will occur 3 times

Reply:
Good catch!